### PR TITLE
Bug 1125525 - Overflow of contact name while tapping on the recipient in composer +autoland

### PR DESCRIPTION
**Pull request by ash7594**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ash7594 Ashutosh Nath Agarwal (ash7594)](https://github.com/ash7594) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6dd180c0c02f)

**Comment by try-server-hook** [comment]
[ash7594 Ashutosh Nath Agarwal (ash7594)](https://github.com/ash7594) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fed73327c8c2)

**Comment by try-server-hook** [comment]
[ash7594 Ashutosh Nath Agarwal (ash7594)](https://github.com/ash7594) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b04b0b89b771)

File: shared/style/confirm.css [hunk 1]
```diff
@@ -50,6 +50,7 @@ form[role="dialog"][data-type="confirm"] h1 {
   color: #fff;
   margin: 0;
   padding: 1rem 1.5rem 0;
+  word-wrap: break-word;
```
Comment (julienw) [inline shared/style/confirm.css]: can we have an ellipsis instead ?

Comment (julienw) [inline shared/style/confirm.css]: Mmm I'm not sure an ellipsis would be a good idea after all. Please keep it like this :)

**Comment by try-server-hook** [comment]
[ash7594 Ashutosh Nath Agarwal (ash7594)](https://github.com/ash7594) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f1fdc185ca0d)

**Comment by autolander** [comment]
The pull request could not be applied to the integration branch. Please try again after current integration is complete.

**Comment by autolander** [comment]
The pull request could not be applied to the integration branch. Please try again after current integration is complete.

**Comment by julienw** [comment]
landed in 229efc18377cd203be97d92995175c08e06307ad

**Pull request closed without merging**